### ISSUE TITLE
Can i invert RGB image to Meter Depth Image?

### ISSUE THREAD

**Issue by HappySalamander** [LiheYoung/Depth-Anything#77]
Thanks for your great job!

Using your model, I want to create a dataset that replaces the Depth image of RGBD Camera.

It seems that the value of the existing depth image is calculated in meters.

Therefore, it consists of values between 0 and 4, and it seems to be segmented and has values for each area.

However, the Depth image that went through your model looks good, but it has a value between 0 and 255. I want to know how I can get this to be generated as a depth image in meters.

Thank you.

**Comment by LiheYoung** [comment]
Hi, the pre-trained models [here](https://github.com/LiheYoung/Depth-Anything/tree/main/metric_depth) produce depth values in meters.

**Comment by Eyshika** [comment]
Why are there 2 separate pretrained models for indoor and outdoor ? Which one do we use for a completely new video/image ? @LiheYoung Also it looks like its just using zoe depth for metric depth calculation, so is it safe to say to use ZoeDepth only ?

**Comment by akebest** [comment]
I have a question: Why can monocular images produce absolute depth, given that they lack stereoscopic vision and reference objects with known sizes? Has the reliability of their absolute depth been tested?"

**Comment by puyiwen** [comment]
> [您好，这里的](https://github.com/LiheYoung/Depth-Anything/tree/main/metric_depth)预训练模型会生成以米为单位的深度值。

Hi, I want to use Depth-anything pretrained model to train my own rgb datasets, and I get 0-697.35 float value. I use the model output value as my dataset gt, to train the relative depth estimation model. However, the model does not converge. Then I tried normalizing the values ​​to 0-1 for training, but found that it still did not converge. Am I wrong to do this? Can you help me answer this question? Thank you very much!! Looking forward to your reply.
